cax/tasks/filesystem.py

```diff
@@ -163,6 +163,108 @@ def each_run(self):
 
             break
 
+
+class AddSize(Task):
+
+    """ Evaluate the file size of a raw data and check if
+        the number of files in the raw data directory match
+        with the number of events recorded
+    """
+    def __init__(self):
+        Task.__init__(self)
+
+    def each_run(self):
+
+        # Check of data info
+        if 'data' not in self.run_doc:
+           self.log.debug("Data not found Name: %s " %(self.run_doc['name']) )
+           return
+
+        # Check of Trigger info
+        if 'trigger' not in self.run_doc:
+           self.log.debug("Trigger not found Name: %s " %(self.run_doc['name']) )
+           return
+
+        run_number = self.run_doc['number']
+        run_name = self.run_doc['name']
+
+
+        trigger = self.run_doc['trigger']
+        nevents = trigger['events_built']
+
+        for data_doc in self.run_doc['data']:
+
+            _host = data_doc['host']
+            _location = data_doc['location']
+            _type = data_doc['type']
+            _status = data_doc['status']
+
+            try:
+                if _type ==  "raw" and _status == "transferred":
+                    self.log.debug("host: %s  location: %s  type: %s" %(_host,_location,_type ) )
+
+
+                    # Check if the number of files match with the number of events
+                    completeness = False
+                    raw_size = 0
+                    nfiles = len(os.listdir(_location))-4
+                    ents = int(nevents)/1000.
```

Comment:
@lucrlom As you mentioned, the number of events is not constant throughout our data taking and this will cause issues here.  Is this check really necessary?  Shouldn't the status check of ```transferred``` above be sufficient?  If not, this points to another bigger problem and we shouldn't be tagging this run as ```transferred```.  Have you actually observed any ```transferred``` runs failing this check?